### DISCUSSION TITLE
Notice Editor Model Examples (Visual, Conceptual and Physical)

### DISCUSSION THREAD

**Discussion by jbarbasgh** [OP-TED/eForms-SDK#375] [Q&A] [answered]
Hi,

Is it possible to make available some examples of a Visual Model (JSON), an Intermediary Conceptual Model (JSON) and the correspondent Physical Model (XML)?
I know that this objects may be different for each implementation, nevertheless I think this could be helpful to better understand the differences that each model has and also could be used to test our implementations.
If this information is already available, please let me know where I can find it.

Thank You.

**Comment by rouschr** [comment, marked as answer]
The editor is showing these, I recommend you play around with one of the smaller notice sub types like X02:
https://github.com/OP-TED/eforms-notice-editor
There are also older questions in this forum where things are explained, search for "model", for example:
https://github.com/OP-TED/eForms-SDK/discussions/306
